cdn/dev/keyboard-search/search.css

```diff
@@ -465,6 +465,10 @@ html[data-platform='unknown'] .download.download-unknown {
   display: block;
 }
 
+.download.download-developer {
```

Comment:
Could you explain why this doesn't use the pattern `html[data-platform='developer']` like above?

Reply:
The `html[data-platform]` attribute is the operating system, so it would be `windows` and not `developer`.